src/test/java/ecdar/utility/StringHelperTestSymbolToUnicode.java

```diff
@@ -0,0 +1,44 @@
+package ecdar.utility;
+
+import ecdar.utility.helpers.StringHelper;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import java.util.Arrays;
+import java.util.Collection;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@RunWith(Parameterized.class)
+public class StringHelperTestSymbolToUnicode {
+    private String inputString;
+    private String expectedResult;
+
+    // Each parameter should be placed as an argument here
+    // Every time runner triggers, it will pass the arguments
+    // from parameters we defined in primeNumbers() method
+
+    public StringHelperTestSymbolToUnicode(String inputString, String expectedResult) {
+        this.inputString = inputString;
+        this.expectedResult = expectedResult;
+    }
+
+    @Parameterized.Parameters
+    public static Collection input() {
+        return Arrays.asList(new Object[][]{
+                {"2 >= 4", "2 \u2265 4"},
+                {"2 <= 4", "2 \u2264 4"},
+                {"2 == 4", "2 == 4"},
+                {"2 > 4", "2 > 4"},
+                {"2 < 4", "2 < 4"}
+        });
+    }
+
+    @Test
+    public void testStringHelperConversionToUnicode() {
+        System.out.println(inputString + " bliver sammenlignet med " + expectedResult);
```

Comment:
I think you forgot to delete this line.
You should not print anything in test cases.